src/objects/strategy-cards/strategy-card-border.js

```diff
@@ -81,21 +81,32 @@ class StrategyCardBorder extends Border {
         return this._ui;
     }
 
+    /**
+     * (Re)position the UI on the players desk. The <code>index</code> is used for
+     * the z-offset in case of multiple strategy card UIs on the same desk.
+     *
+     * @param {Integer} index Position of the UI on the stack on the players desk
+     */
+    positionUi(index) {
+        this._ui.position = this._desk.localPositionToWorld({
+            x: 30,
+            y: 0,
+            z: 20 + index * -1,
```

Comment:
You could also take a look at src/lib/ui/tabbed-panel, but this is good

Reply:
I considered that. In the end I wanted to focus the users attention more to work the strategy cards one by one in play order.